doc/getting-started/overview_dev.rst

```diff
@@ -101,20 +101,21 @@ ACRN offers three types of VMs:
 
 * **Post-launched User VMs**: These VMs typically share hardware resources via
   the Service VM and Device Model. They can also access hardware devices
-  directly if they've been configured as passthrough devices. Unlike
-  pre-launched VMs, you can change the configuration at runtime. They are
-  well-suited for non-safety applications, including human machine interface
-  (HMI), artificial intelligence (AI), computer vision, real-time, and others.
+  directly if they've been configured as passthrough devices. The configuration
+  of a post-launched VM can be static (defined at build time) or dynamic
+  (defined at runtime without rebuilding ACRN). They are well-suited for
```

Comment:
While a true statement, we have no practical information about the use case or documented details of how to define, launch, and manage  these "Dynamic VMs".  This is the first time I've seen this distinction pop up and we'd need yet another tutorial/guide on how do this. @NanlinXie